Configuration/PyReleaseValidation/python/relval_nano.py

```diff
@@ -449,6 +461,8 @@ def subnext(self):
 workflows[_wfn()] = ['lepTrackInfoNANOdata140Xrun3', ['MuonEG2024MINIAOD14.0', 'lepTrackInfoNANO_data14.0']]
 workflows[_wfn()] = ['ScoutingNANOdata140Xrun3', ['ScoutingPFRun32024RAW14.0', 'scoutingNANO_data14.0']]
 workflows[_wfn()] = ['ScoutingNANOwithPromptdata140Xrun3', ['ScoutingPFMonitor2024MINIAOD14.0', 'scoutingNANO_withPrompt_data14.0']]
+workflows[_wfn()] = ['L1ScoutingNANOdata140Xrun3', ['L1Scouting2024RAW14.0', 'l1ScoutingNANO_data14.0']]
+workflows[_wfn()] = ['L1ScoutingSelectionNANOdata140Xrun3', ['L1ScoutingSelection2024RAW14.0', 'l1ScoutingSelectionNANO_data14.0']]
```

Comment:
Looks like these two lines overflow the WF counter. 

My naive suggestion for now would be to move the L1Scounting to a new series (2500.28x). I also leave it for the other to comment.

Reply:
@patinkaew -- please, have a look at https://github.com/cms-sw/cmssw/pull/48459